common/common.go

```diff
@@ -486,3 +487,21 @@ func GetExitStatus(err error) (int, error) {
 	}
 	return -1, err
 }
+
+// GetArch returns the current architecture. It returns runtime.GOARCH if not on a arm device
+// and armv6l, armv7l, arm64 otherwise
+func GetArch() string {
+	arch := runtime.GOARCH
+	if arch == "arm" {
+		utsName := &syscall.Utsname{}
+		syscall.Uname(utsName)
+		arch = ""
+		for _, b := range utsName.Machine {
+			if b == 0 {
+				break
+			}
+			arch += string(b)
+		}
+	}
+	return arch
```

Comment:
First I wanted to use this, but I don't know a way of getting the arm-flavor (5,6,7...) from go. I think, I could use the runtime/internal/sys GoarchArm constant, but I can not use internal packages. If you know a way of getting this information I would be very happy to use the ToAppcOSArch function.